app/models/task.rb

```diff
@@ -5,4 +5,6 @@ class Task < ApplicationRecord
 
   validates :title, presence: true, length: { minimum: 5 }
   validates :body, presence: true
+
+  enum :status, { in_progress: 0, done: 1 }
```

Comment:
I think we should have default status like `pending` or `queued`  which tells it is in not picked up yet.